### ISSUE TITLE
Trouble passing VectorVectorInt

### ISSUE THREAD

**Issue by decrispell** [libigl/libigl#652]
I'm having trouble passing arguments of type pyigl.VectorVectorInt into pyigl.boundary_loop().  I get a strange "incompatible function arguments" error that lists the argument types used as one of the available options (See example below).   Has anybody seen something similar?  This used to work but I haven't been able to pin down the breaking change yet.

```
import pyigl
F = pyigl.eigen.MatrixXi.Zero(10,3)  # dummy input
all_bnd = pyigl.VectorVectorInt()
pyigl.boundary_loop(F, all_bnd)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-6-4dab35c196af> in <module>()
----> 1 pyigl.boundary_loop(F, all_bnd)

TypeError: boundary_loop(): incompatible function arguments. The following argument types are supported:
    1. (F: pyigl.eigen.MatrixXi, L: pyigl.eigen.MatrixXi) -> None
    2. (F: pyigl.eigen.MatrixXi, L: pyigl.VectorVectorInt) -> None
    3. (F: pyigl.eigen.MatrixXi, L: pyigl.VectorInt) -> None

Invoked with: 0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0
0 0 0, <pyigl.VectorVectorInt object at 0x7f958c144660>
```

**Comment by decrispell** [comment]
Additional details that I should have included in the original post:   Calling the `pyigl.boundary_loop(MatrixXi, MatrixXi)` version seems to work fine, but both `pyigl.boundary_loop(MatrixXi, VectorVectorInt)` and the `pyigl.boundary_loop(MatrixXi, VectorInt)` return similar "incompatible function arguments" errors.

That is the reason that I suspect the problem has something to do with the pybind11 `bind_vector` calls [here](https://github.com/libigl/libigl/blob/d4b67260855fe901541f2949ff21819147c08a4d/python/modules/py_typedefs.cpp#L23-L24) .   But that's just speculation at this point, as I haven't found any problem with them.

I'm using the current libigl head (d4b67260855fe901541f2949ff21819147c08a4d) and python 2.7.  Can anyone reproduce?

**Comment by skoch9** [comment]
I could reproduce it and work on a fix currently. There is already a warning during the building process about this...

**Comment by skoch9** [comment]
Your problem should be solved with pull request #654.

**Comment by decrispell** [comment]
#654 did indeed fix the problem.  Thank you!
